src/deadline/client/cli/_groups/asset_group.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+"""
+All the `deadline asset` commands:
+    * snapshot
+    * upload
+    * diff
+    * download
+"""
+
+import click
+
+from .._common import _handle_error
+
+
+@click.group(name="asset")
+@_handle_error
+def cli_asset():
+    """
+    Commands to work with AWS Deadline Cloud Job Attachments.
+    """
+
+
+@cli_asset.command(name="snapshot")
+@click.option("--root-dir", help="The root directory to snapshot.")
+@_handle_error
+def asset_snapshot(**args):
+    """
+    Creates manifest of files specified root directory.
+    """
+    click.echo("snapshot taken")
+
+
+@cli_asset.command(name="upload")
+@click.option("--manifest", help="The manifest to ")
+@_handle_error
+def asset_upload(**args):
+    """
+    Preupload files before for a render with specified manifest.
+    """
+    click.echo("upload done")
+
+
+@cli_asset.command(name="diff")
+@_handle_error
+def asset_diff(**args):
+    """
+    Check file differences of a directory since last snapshot.
```

Comment:
Can we expand on this a bit? Maybe give an example of what the output would be?

If we're still not sure on that, maybe we can have a placeholder with a TODO comment?